src/Aspire.Hosting.Docker/api/Aspire.Hosting.Docker.cs

```diff
@@ -0,0 +1,533 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Aspire.Hosting.Docker
+{
+    public static partial class DockerComposePublisherExtensions
+    {
+        public static void AddDockerCompose(this IDistributedApplicationBuilder builder, string name, System.Action<DockerComposePublisherOptions>? configureOptions = null) { }
+    }
+
+    public sealed partial class DockerComposePublisherOptions : Publishing.PublishingOptions
+    {
+        public string? DefaultContainerRegistry { get { throw null; } set { } }
+
+        public string? ExistingNetworkName { get { throw null; } set { } }
+    }
+}
+
+namespace Aspire.Hosting.Docker.Resources
```

Comment:
Same here @eerhardt. This is basically the CDK for docker compose files.

Reply:
This package is will be prerelease:

https://github.com/dotnet/aspire/blob/b5752c261dcc363cd78980057ed72785a7119390/src/Aspire.Hosting.Docker/Aspire.Hosting.Docker.csproj#L6